src/metabase/task/sweep_query_analysis.clj

```diff
@@ -0,0 +1,90 @@
+(ns metabase.task.sweep-query-analysis
```

Comment:
<putting on my hat as the local Marginalia gremlin>

It'd be nice to have a ns-level docstring that summarized what the job does, how you define stale and orphan cards, etc.

Reply:
Oh man, you're right but I'm too eager right now and will put on my technical writing hat [tomorrow](https://github.com/metabase/metabase/issues/45720).